### PR TITLE
opentracing upgrade from 0.31 to 0.33

### PR DESCRIPTION
**Pull request by suman-ganta**
upgraded opentracing from 0.31 to 0.33

### CODE REVIEW

**Comment by Scottmitch** [comment]
@servicetalk-bot - test this please

File: servicetalk-opentracing-http/src/main/java/io/servicetalk/opentracing/http/AbstractTracingHttpFilter.java [hunk 1]
```diff
@@ -116,8 +126,20 @@ protected boolean isError(final HttpResponseMetaData metaData) {
...
+
+        private void closeAll() {
+            try {
+                currentScope.close();
+            } finally {
+                span.finish();
+            }
+        }
+
+        protected final Span getSpan() {
```
Comment (Scottmitch) [inline servicetalk-opentracing-http/src/main/java/io/servicetalk/opentracing/http/AbstractTracingHttpFilter.java:141]: can you make this package private (least access required)?

File: servicetalk-opentracing-http/src/main/java/io/servicetalk/opentracing/http/TracingHttpRequesterFilter.java [hunk 2]
```diff
@@ -133,8 +134,26 @@ private ScopeTracker newTracker(final HttpRequestMetaData request) {
...
+        try {
+            tracer.inject(span.context(), formatter, request.headers());
+            return new ScopeTracker(scope, span);
+        } catch (Throwable cause) {
+            handlePrematureError(span, scope);
+            throw cause;
+        }
+    }
+
+    private void handlePrematureError(Span span, Scope scope) {
```
Comment (Scottmitch) [inline servicetalk-opentracing-http/src/main/java/io/servicetalk/opentracing/http/TracingHttpRequesterFilter.java:148]: can you make this method `static`?

File: servicetalk-opentracing-http/src/test/java/io/servicetalk/opentracing/http/TracingHttpRequesterFilterTest.java [hunk 3]
```diff
@@ -158,33 +158,34 @@ public void testInjectWithParent() throws Exception {
...
+                        assertEquals(GET.name(), lastFinishedSpan.tags().get(HTTP_METHOD.getKey()));
+                        assertEquals(requestUrl, lastFinishedSpan.tags().get(HTTP_URL.getKey()));
+                        assertEquals(OK.code(), lastFinishedSpan.tags().get(HTTP_STATUS.getKey()));
+                        assertFalse(lastFinishedSpan.tags().containsKey(ERROR.getKey()));
 
-                    verifyTraceIdPresentInLogs(stableAccumulated(1000), requestUrl, serverSpanState.traceId,
+                        verifyTraceIdPresentInLogs(stableAccumulated(1000), requestUrl, serverSpanState.traceId,
                             serverSpanState.spanId, serverSpanState.parentSpanId, TRACING_TEST_LOG_LINE_PREFIX);
-                }
+                    }
```
Comment (Scottmitch) [inline servicetalk-opentracing-http/src/test/java/io/servicetalk/opentracing/http/TracingHttpRequesterFilterTest.java:188]: can you add the following:

```java
 finally {
  clientSpan.finish();
}
```

**Comment by suman-ganta** [comment]
@servicetalk-bot - test this please

**Comment by Scottmitch** [comment]
@servicetalk-bot - test this please

**Comment by suman-ganta** [comment]
@idelpivnitskiy - can it be merged?

**Pull request merged**